test/playwright.config.js

```diff
@@ -0,0 +1,44 @@
+import { defineConfig, devices } from '@playwright/test'
+import * as dotenv from 'dotenv'
+dotenv.config()
+
+const { BASE_URL } = process.env
+
+module.exports = defineConfig({
+  testDir: './src',
+  timeout: 1000 * 120,
+  expect: {
+      timeout: 1000 * 120
+  },
+  forbidOnly: false,
+  retries: 0,
+  workers: 2,
+  reporter: 'list',
+  use: {
+    actionTimeout: 1000 * 30,
+    navigationTimeout: 1000 * 30,
+    baseURL: `${BASE_URL}`,
+    headless: true,
+    trace: 'on-first-retry',
+    screenshot: 'only-on-failure',
+    video: 'retain-on-failure',
+    launchOptions: {
+      slowMo: 1000,
+      // logger: {
```

Comment:
Is there a use-case for when we'd want to uncomment these lines? If so, is that documented?

Reply:
Yes, logging can be helpful for debugging tests. I can add comments in the config file and make a note in the testing docs.